app/frontend/src/components/forms/PrintOptions.vue

```diff
@@ -0,0 +1,221 @@
+    files() {
+      return this.templateForm.files;
+    },
+  },
+  methods: {
+    ...mapActions('notifications', ['addNotification']),
+    async printBrowser() {
+      this.dialog = false;
+      // Setting a timeout to allow the modal to close before opening the windows print
+      setTimeout(() => {
+        window.print();
+      }, 500);
+    },
+    toTextObject(file) {
+      return new Promise((resolve, reject) => {
+        const reader = new FileReader();
+        reader.readAsText(file);
+        reader.onload = () => resolve(reader.result);
+        reader.onerror = (error) => reject(error);
+      });
+    },
+    splitFileName(filename = undefined) {
+      let name = undefined;
+      let extension = undefined;
+
+      if (filename) {
+        const filenameArray = filename.split('.');
+        name = filenameArray.slice(0, -1).join('.');
+        extension = filenameArray.slice(-1).join('.');
+      }
+
+      return { name, extension };
+    },
+    fileToBase64(file) {
+      return new Promise((resolve, reject) => {
+        const reader = new FileReader();
+        reader.readAsDataURL(file);
+        reader.onload = () => resolve(reader.result.replace(/^.*,/, ''));
+        reader.onerror = (error) => reject(error);
+      });
+    },
+    getDispositionFilename(disposition) {
+      return disposition ? disposition.substring(disposition.indexOf('filename=') + 9) : undefined;
+    },
+    createDownload(blob, filename = undefined) {
+      const url = window.URL.createObjectURL(blob);
+      const a = document.createElement('a');
+      a.style.display = 'none';
+      a.href = url;
+      a.download = filename;
+      a.click();
+      window.URL.revokeObjectURL(url);
+      a.remove();
+    },
+    async generate() {
+      try {
+        this.loading = true;
+        const outputFileType = 'pdf';
+        let content = '';
+        let contentFileType = '';
+        let outputFileName = '';
+
+        content = await this.fileToBase64(this.templateForm.files);
+        contentFileType = this.templateForm.contentFileType;
+        outputFileName = this.templateForm.outputFileName;
+
+        const body = this.createBody(
+          content,
+          contentFileType,
+          outputFileName,
+          outputFileType
+        );
+
+        // Submit Template to CDOGS API
+        const response = await formService.docGen(this.submissionId, body);
+
+        // create file to download
+        const filename = this.getDispositionFilename(
+          response.headers['content-disposition']
+        );
+
+        const blob = new Blob([response.data], {
+          type: 'attachment',
+        });
+
+        // Generate Temporary Download Link
+        this.createDownload(blob, filename);
+        this.addNotification({
+          message: 'Parsed successfully',
+          ...NotificationTypes.SUCCESS,
+        });
+      } catch (e) {
+        this.addNotification({
+          message: 'Failed upload Template',
```

Comment:
An error occurred while trying to generate your document
Or something like that?